src/styles/components/sequence.scss

```diff
@@ -2,14 +2,17 @@
 @import "../colours";
 
 .sequence {
-  font-family: $font-family-monospace;
   display: flex;
-  flex-wrap: wrap;
-  &__chunk {
-    margin: ($global-margin/2);
-    &__counter {
-      text-align: right;
-      min-height: 1.6rem;
+  align-items: flex-start;
+  &__sequence {
+    font-family: $font-family-monospace;
+    display: flex;
+    flex-wrap: wrap;
+    &__chunk {
+      margin: ($global-margin/2);
```

Comment:
I think CSS-linters would like to see a space before and after operators:

    margin: ($global-margin / 2);